### PR TITLE
Help messages

### PR DESCRIPTION
**Pull request by Tauras-Rim**
-Merged SuggestionsService comparison methods into one method
-Deleted unused properties in index.cshtml.cs
-Adjusted code in index.cshtml

### CODE REVIEW

File: Energy Saver/Pages/Index.cshtml [hunk 1]
```diff
@@ -74,20 +74,25 @@
         <body>
             <div>
                 @{
-                    string[] comparison = new string[4];
+                    List<string> taxList = new List<string>();
```
Comment (priolette) [inline Energy Saver/Pages/Index.cshtml]:
```suggestion
                    List<string> taxList = new List<string> { "Gas", "Electricity", "Water", "Heating" };
```

**Review by priolette — CHANGES_REQUESTED** [review]
Great changes all around but please check out my suggestions regarding the list in Index.cshtml file

File: Energy Saver/Pages/Index.cshtml [hunk 2]
```diff
@@ -74,20 +74,25 @@
...
+                    taxList.Add("Gas");
+                    taxList.Add("Electricity");
+                    taxList.Add("Water");
+                    taxList.Add("Heating");
+                }
+                @for(int i = 0; i < Model.taxComparison.Count(); i++)
+                {
+                    if(Model.taxComparison[i] > 0)
+                    {
+                        <p class="text-color" style="margin-top: 40px">@taxList[i] consumption this month compared to last month: <span class="text-negative")">+@Model.taxComparison[i]%</span></p>
```
Comment (Leminode) [inline Energy Saver/Pages/Index.cshtml]: should probably put @taxList[i] inside curly brackets to avoid ambiguity

**Review by Leminode — APPROVED** [review]
Good job

**Comment by priolette** [comment]
Approved

**Review by priolette — APPROVED** [review]
Approved

**Pull request merged**